### ISSUE TITLE
[Bug] - deployed site not updating with latest changes

### ISSUE THREAD

**Issue by jdwilkin4** [freeCodeCamp/Developer_Quiz_Site#976]
### What bug did you find in the codebase?

Some design changes were added but that haven't shown up on the live site.
The design changes work fine locally when you `pnpm run start`
It also works locally, when you do a local build and run `pnpm run preview`

But on the live site, the design changes don't show up.
After digging deeper, it looks like the last few PR changes are not reflected in the site.

On the live site, the total number of random questions shows 1560 but locally the total number of questions shows 1570.

It is possible something went from the the previous configuration changes that happened a few months back. 
So that might be a good place to start looking.

**Comment by shootermv** [comment]
I cans see pages-build-deployments task no more running

**Comment by jdwilkin4** [comment]
That job was triggered by me a few hours ago.

<img width="228" alt="Screenshot 2024-07-01 at 3 22 53 AM" src="https://github.com/freeCodeCamp/Developer_Quiz_Site/assets/67210629/93140ca7-14d2-44b2-be9f-a2a89beaa305">


But I have found the issue.

If you look closely at the link to where it is deploying to, it is incorrect
<img width="704" alt="Screenshot 2024-07-01 at 3 23 25 AM" src="https://github.com/freeCodeCamp/Developer_Quiz_Site/assets/67210629/7dc57a5f-657a-4f7e-a188-8bec634892fb">

The last deployment for the actual developerquiz.org was back in April

<img width="1282" alt="Screenshot 2024-07-01 at 3 23 44 AM" src="https://github.com/freeCodeCamp/Developer_Quiz_Site/assets/67210629/c6837bd9-fe0f-403c-a49d-decaee60fa89">

ever since then, it has been deploying to `https://opensource.freecodecamp.org/Developer_Quiz_Site/`

So it broke somewhere around that time

**Comment by jdwilkin4** [comment]
I am going to remove the help wanted label because the team is looking into setting it back to the correct url

**Comment by raisedadead** [comment]
This issue (rather confusion) is coming from the conflicting nature of the deployment and settings.

I see that there is action configured for pages deployment, that abstracts deployment to a third-party package. At the same time I see that there is default setting which uses `gh-pages` branch based deployment.

The former will "upload" files from the `dist` and the later will use a GitHub native feature (you see it as the `pages-build-deployment` action (not the same as us).

In simple terms this is a bit convoluted and conflicting approach that can break.

I am working to move this over to Cloudflare Pages. Which is simpler to manage from our end. I will have a PR shortly.

**Comment by jdwilkin4** [comment]
Awesome, thanks @raisedadead for looking into this.

I am kind of surprised we haven't run into this issue before, but I am glad we are moving to cloudflare pages.

**Comment by shootermv** [comment]
I know that vercel cloud has very simple way to integtrate with github, may it is something worth to consider to move to...
